### ISSUE TITLE
visual clarity of options

### ISSUE THREAD

**Issue by annamorphism** [DDMAL/CantusDB#441]
Old Cantus has a little line between View/Edit, which (in my opinion) makes it clearer they lead to separate "tabs" 
![image](https://user-images.githubusercontent.com/67451875/204898055-a20072a0-107d-4c96-91c4-eb722f2ae128.png)
![image](https://user-images.githubusercontent.com/67451875/204897861-a51493dd-1275-4c97-8e3a-400130050a8a.png)

also I think this layout is confusing:
![image](https://user-images.githubusercontent.com/67451875/204901376-d1291c9d-4eb8-475d-acb9-6e475cc6caa1.png)
Probably "save" and "delete" should be more prominent, and above the edit-syllabification link (which shouldn't have a colon after it.)

**Comment by annamorphism** [comment]
also is there a way to not have the "Edit" tab show up for unauthorized people? it's annoying to try to edit something and then be sent to a 403.

**Comment by jacobdgm** [comment]
> also is there a way to not have the "Edit" tab show up for unauthorized people? it's annoying to try to edit something and then be sent to a 403.

wait, this is happening? I thought this was working properly... let me investigate - we should probably write a test for this to make sure it doesn't stop working again.

**Comment by jacobdgm** [comment]
> Old Cantus has a little line between View/Edit

i.e. the horizontal one that runs below "View" and above "Edit" in your screenshot? I think that the horizontal lines look a bit messy while the vertical bar between the two is simpler and more elegant (but go figure, I was the one to set it up). Let me fiddle with this and see if I can find a way that make it clear there are two separate tabs while still looking neat.

Re: syllabification prompt, I think I can see why the options are in that order: presumably you have to edit the syllabification before saving, not after. And you can only view your changes once you've saved the chant, so the "open chant page" link going after the buttons also kind of makes sense. Maybe it would be most ideal if the buttons were right at the bottom? That's where I'd expect to find them, I think...

**Comment by jacobdgm** [comment]
If we have a view/edit toggle at the top, we probably don't need the "open chant page" at the bottom anyways. It's not on OldCantus as far as I can tell, so I wonder why/how it ended up there in the first place...

**Comment by annamorphism** [comment]
> If we have a view/edit toggle at the top, we probably don't need the "open chant page" at the bottom anyways. 

I thought this too, but wondered if there was a reason both had been put in...

**Comment by annamorphism** [comment]
> > Old Cantus has a little line between View/Edit
> 
> i.e. the horizontal one that runs below "View" and above "Edit" in your screenshot? I think that the horizontal lines look a bit messy while the vertical bar between the two is simpler and more elegant (but go figure, I was the one to set it up). Let me fiddle with this and see if I can find a way that make it clear there are two separate tabs while still looking neat.

I think part of it is that the tabs were set up "backwards" in the first place (I know Debra and I talked about this once, ages ago...) I think the active tab should be the one with the rectangle around it, so it is "in front".  Unless you want to get into some sort of fancy shading thing like Wikipedia...
![image](https://user-images.githubusercontent.com/67451875/205337847-b3efe55b-91e5-4b9f-9898-2a1ce871c0d4.png)
 

> Re: syllabification prompt, I think I can see why the options are in that order: presumably you have to edit the syllabification before saving, not after. And you can only view your changes once you've saved the chant, so the "open chant page" link going after the buttons also kind of makes sense. Maybe it would be most ideal if the buttons were right at the bottom? That's where I'd expect to find them, I think...

I think there needs to be more space between the syllabification--which is something to think about as you are entering/inspecting Volpiano--and the buttons, which apply to the whole chant. Actually it is a little weird to me that the edits fields display in a different order from the chant-view fields---why is all the stuff about folio/sequence/etc moved underneath the text part?  With the text entry lower down one could have [text--volpiano--edit syllabification button--preview  text and music] all grouped together.

**Comment by annamorphism** [comment]
> wait, this is happening? I thought this was working properly... let me investigate - we should probably write a test for this to make sure it doesn't stop working again.

Hmm... I think it correctly doesn't display if I'm not logged in. If I am logged in, but I'm not attached to the source in the right way (including a few I worked on!) it displays but sends me to an access-denied page.  (This also means, since I am not a Debra in this Cantus, that I can't add myself to edit any sources I didn't create. Maybe that is a good thing, or maybe it creates more work for admins--probably worth reviewing at some point what each level of editor is restricted to.)
